### ISSUE TITLE
Is this course based on axa-group nlp.js

### ISSUE THREAD

**Issue by macasas** [jesus-seijas-sp/nlp-course#7]
Hola Jesus

Sorry to contact you here but I can find no other way. 

Is this course based on the axa-group nlp.js node module?  
https://github.com/axa-group/nlp.js

Thanks

**Comment by jesus-seijas-sp** [comment]
Hello @macasas ,
This course uses NLP.js, but also Brain.js and Tensorflow.js.
Is not a course about chatbots, it's about Conversational AI, and explain all kind of metrics (confusion matrix, accuracy, precision, recall...), and also how to develop a perceptron and a dense neural network from scratch. It's in udemy.

**Comment by macasas** [comment]
Thank you, I found the course on Udemy, and in your preview it look like the axa nlp.js  but I want to confirm this, because nlp is a very generic term, and node modules are well known for having similar names.  It used to be node-nlp, then nlp.js, so I want to be sure it is the same module. 

if I search for nlp.js, node nlp or similar, I end up with axa-group nlp.js but its not clear if this is the only one.

You also have a repo called nlp.js.  Is this the same as axa-group?  Is this the node module you are using in the course?

Please confirm which NLP.js

**Comment by jesus-seijas-sp** [comment]
The course is about Natural Language Processing using Javascript. To achieve that, the classifiers are done with different technologies: Brain.js, Tensorflow.js, Microsoft LUIS, NLP.js neural network (one of the libraries of NLP.js) and creating a perceptron and neural network from scratch without any dependency. Then those classifiers are compared.
The course includes usage of lot of libraries from axa-group/nlp.js, like the normalizers, tokenizers, stemmers, neural network, benchmark, confusion matrix...
Finally, the last module of the course is about creating a Q&A web chatbot using NLP.js

About the node-nlp and NLP.js thing: axa-group/nlp.js name was always NLP.js, but in npm this name was already taken, so the library was published as node-nlp. Versions 1.x, 2.x and 3.x were monolithic, so the name of the package was node-nlp. Since version 4, this is no longer monolithic, and it's splitted into more than 80 libraries (example, 41 libraries one for each language). One of this 80+ libraries is node-nlp, that is still there for retrocompatibility with people that used previous versions or for people that feels more confortable with "having it all in a library" approach.

**Comment by macasas** [comment]
Okay, muchas gracias Jesus. Me parece muy bien, exactamente lo que busco. :-)
